node/src/chain_spec/pangoro.rs

```diff
@@ -255,8 +255,8 @@ pub fn genesis_config() -> ChainSpec {
 		None,
 		Some(properties()),
 		Extensions {
-			relay_chain: "rococo".into(), // You MUST set this to the correct network!
```

Comment:
@hackfisher @jiguantong Please help check this field.

Reply:
Right